aviary/modules/binning/envs/comebin-gpu.yaml

```diff
@@ -0,0 +1,9 @@
+channels:
+  - pytorch
+  - nvidia
+  - conda-forge
+  - bioconda
+dependencies:
+  - comebin=1.0.*
+  - pytorch

```

Comment:
Is pinning `pytorch` to a version not needed?

Reply:
Do you recommend?

Reply:
Yeah, makes user installations more consistent and easier to debug. We might not know that a new pytorch version breaks this yaml until someone else points it out. Downside is we miss out on updates unless we manually update the versions pins ourselves, but worth it imo

Reply:
Note that our GPUs are too new for the version of pytorch that comebin is compatible with. So I can't test the GPU usage. But the env installs fine and CPU fallback works even in the `-gpu` env.

```
NVIDIA H100 80GB HBM3 MIG 1g.10gb with CUDA capability sm_90 is not compatible with the current PyTorch installation.
The current PyTorch install supports CUDA capabilities sm_37 sm_50 sm_60 sm_61 sm_70 sm_75 sm_80 sm_86 compute_37.
```

Reply:
Yooo when did you get H100s?? Nice

Maybe pinning to a version is a bad idea for pytorch as gpu architecture is really the more difficult part

Reply:
Not sure about this. I get the feeling that conda doesn't check for gpu compatibility when choosing versions.
For instance, comebin is only compatible with an old version of pytorch that isn't compatible with our gpus, but the env installs fine with no errors or warnings.

Reply:
I'll leave it pinned for now and see if anyone else has issues.